app/controllers/merchant_site/items_controller.rb

```diff
@@ -1,6 +1,36 @@
 module MerchantSite
   class ItemsController < ApplicationController
     def index
+      @merchant = Merchant.find(params[:merchant_id])
+    end
+
+    def show
+      @merchant = Merchant.find(params[:merchant_id])
+      @item = Item.find(params[:id])
+    end
+
+    def edit
+      @merchant = Merchant.find(params[:merchant_id])
+      @item = Item.find(params[:id]) 
+    end
+
+    def update
+      @merchant = Merchant.find(params[:merchant_id])
+      @item = Item.find(params[:id])
+      @item.update(item_params)
+      # require 'pry' ; binding.pry
+      if params[:status]
```

Comment:
Updating params with the enable/disable status buttons. There's no default value (NULL) but when we click the buttons it changes them in the DB